miasm2/expression/expression.py

```diff
@@ -511,6 +522,9 @@ def __new__(cls, name, size=32):
     def __str__(self):
         return str(self.__name)
 
+    def str_sub_expr(self):
```

Comment:
Also, I'm not sure if this is more a Miasm internal, and we probably don't want to expose it in the public API. So maybe a '_str_sub_expr' could be fine.